examples/plotting/200_cell_transitions.ipynb

```diff
@@ -101,38 +101,11 @@
      "name": "stdout",
```

Comment:
<p>Analogously, we can compute the likelihoods a cell originates from by setting <code>source_groups="cell_type"</code>,...., <code>forward=False</code>:</p><p> </p><p>[Please make example]  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>